src/app/components/article/ArticleSectionChartList.tsx

```diff
@@ -0,0 +1,52 @@
+import { ArticleChartList, ArticleList } from 'app/utils/mockUp';
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+
+interface ArticleSectionChartListProps {
+  articleList?: ArticleChartList[];
+}
+
+export const groupChartArticle = (groupingUnitCount: number) => (groupedArticles: ArticleList[][], article: ArticleList, idx: number) => {
+  if (idx % groupingUnitCount === 0) {
+    groupedArticles.push([article]);
+  } else {
+    groupedArticles[groupedArticles.length - 1].push(article);
+  }
+  return groupedArticles;
+};
+
+export const ArticleSectionChartList: React.FunctionComponent<ArticleSectionChartListProps> = (props) => {
+  const { articleList } = props;
+  return (
+    <div className="ArticleChartList_Wrapper">
+      { articleList && articleList
+        .slice(0, 6)
+        .reduce(groupChartArticle(3), [])
+        .map( (groupedArticles, groupIdx) => (
+          <ol className="ArticleChartGroup" start={groupIdx * 4 + 1} key={groupIdx}>
+            { groupedArticles.map((article, idxInGroup) => {
+                const index = groupIdx * 3 + idxInGroup;
+                return (
+                  <li key={String(groupIdx) + idxInGroup} className="Article">
```

Comment:
key는 하나의 배열 안에서만 유일하면 됩니다.

```suggestion
                  <li key={idxInGroup} className="Article">
```